app/tests/unit/components/Form/ProjectFundingAgreementFormSummary.test.tsx

```diff
@@ -204,6 +210,7 @@ describe("The Project Funding Agreement Form Summary", () => {
   it("Only displays the data fields that have changed for an IA form", () => {
```

Comment:
In this one and the similar EP one, you could additionally assert that the payments tracker isn't showing

Reply:
good idea

Reply:
the payments tracker is no longer a thing